services/121-service/src/payments/reconciliation/excel/excel-reconciliation.module.ts

```diff
@@ -4,8 +4,8 @@ import { TypeOrmModule } from '@nestjs/typeorm';
 
 import { ActionsModule } from '@121-service/src/actions/actions.module';
 import { ExcelModule } from '@121-service/src/payments/fsp-integration/excel/excel.module';
-import { ExcelRecociliationController as ExcelReconciliationController } from '@121-service/src/payments/reconciliation/excel/excel-reconciliation.controller';
-import { ExcelRecociliationService as ExcelReconciliationService } from '@121-service/src/payments/reconciliation/excel/excel-reconciliation.service';
+import { ExcelReconciliationController as ExcelReconciliationController } from '@121-service/src/payments/reconciliation/excel/excel-reconciliation.controller';
+import { ExcelReconciliationService as ExcelReconciliationService } from '@121-service/src/payments/reconciliation/excel/excel-reconciliation.service';
```

Comment:
@nielsbom this rename 'as' is no longer necessary

Reply:
If the code will be deleted: feel free to delete the PR, task and branch.

If not: then I don't understand you :-)

Reply:
```suggestion
import { ExcelReconciliationService } from '@121-service/src/payments/reconciliation/excel/excel-reconciliation.service';
```
(same for the other line)